index.html

```diff
@@ -0,0 +1,332 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="new project for microverse">
+    <link rel="stylesheet" href="/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Ubuntu:wght@300&display=swap"
+        rel="stylesheet">
+    <link href="<link rel=" preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Kanit:wght@600&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@200;400&family=Ubuntu:wght@300&display=swap"
+        rel="stylesheet">
+    <title>My portfolio project</title>
+</head>
+
+<body>
+   
+    <header id="navbar">
+        <nav id="menubar">
+            <a href="index.html" id="logo">My Logo</a>
+            <img id="menu" src="/img/menu.svg" alt="menu-button">
+        </nav>
+
+    </header>
+    <main>
+        <section id="mainCont">
+            <h1 id="name">
+                Damian M. <br> Dib Bustros
+            </h1>
+
+            <p id="introduction">
+                Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my
+                work. <br>
+                <br> If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+
+            <div class="buttons" id="start-collaboration">
+                <a href="#" class="submit-button">Start collaboration</a>
+            </div>
+            
+            <br>
+            <br>
+            
+            <div id="geometry">
+                <img id="geometryDiv" src="/img/geometry.svg" alt="geometryDivisor">
+            </div>
+
+
+            <section id="cardBoxes">
+            <div id="projectsTitle">
+                <h2>Projects</h2>
+            </div>
+
+            <section id="gridCont">
+
+                <div class="projectBox">
+                    <div class="boxContainer">
+                        <div class="picture">
+                            <img class="picturesPreview" src="/img/pic1.png" alt="picture1">
+                        </div>
+                        <div class="textBox">
+                            <div class="boxTitle">
+                                Project name goes here
+                            </div>
```

Comment:
- [ ] kindly consider making this project title a hx this will help you  to fulfill the below requirement 
> The project title (e.g., "Tonic") use a header tag hx